PinetimeFlasher.pyw

```diff
@@ -1,13 +1,24 @@
 #!/usr/bin/env python3
-
+import hashlib
+import platform
 import sys
 import os
 import shutil
+import requests
+import json
+import wget
 from pathlib import Path
 from PyQt5.QtCore import *
 from PyQt5.QtWidgets import *
 from PyQt5.QtGui import *
 import pickle
+from typing import List
+
+
+def add_openocd_to_system_path(status_notice: QLabel):
+    openocd_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'openocd', 'bin')
+    os.environ["PATH"] = os.environ["PATH"] + os.pathsep + openocd_path
+    status_notice.setText("OpenOCD downloaded and added to system path.")
```

Comment:
Since this function fires when the main app window is created, do we 'the dumb user'
want to see this when the app starts (and OpenOCD 'portable' is found?)

Reply:
This being ln#21 ...